duckduckgo/Android
# Fix share failure and blank tabs

Task/Issue URL: https://app.asana.com/0/488551667048375/656938381127814 and https://app.asana.com/0/361428290920652/611385370877136
Tech Design URL: N/A

**Description**:
• Removes the manual launching of share intents into tabs allowing live data updates to trigger them. This fixes an issue where sharing from an external app such as Chrome sometimes failed, resulting in a blank tab.
• Clears unused blank tabs

**Steps to test this PR**:
*SHARE TESTS:*

- With the app open, share a url from chrome to our app and ensure the shared url displays
- With the app closes, share a url from chrome to our app and ensure the shared url displays
- Now repeat the above steps with developer settings set to "Don't keep activities" on and "Background process limit" set to one
- Reset the settings changed in the previous step back to defaults

*BLANK TABS TEST:*
1. Open a few webpages in new tabs
1. Now launch the app from the widget multiple times, note that the tabs created in step one are still present and only one duckduckgo home tab is generated
1. Now repeat the previous step this time launching the app from assist

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
